.scalafmt.conf

```diff
@@ -1,5 +1,6 @@
-version = "2.7.5"
+version = "3.0.0-RC2"
```

Comment:
bumped in a preliminary commit to support the sources with significant indentation in the scripted tests